### PR TITLE
Html edited

### PR DESCRIPTION
**Pull request by aminukano585**
can you please make it more visible?

### CODE REVIEW

**Comment by aminukano585** [comment]
consider it done!

**Comment by Timtech4u** [comment]
Cool, you guys are doing great!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (Auwalms) [inline index.html:7]: @Timtech4u @ambat25 @taheermb can you help us link the css and js in this location

**Review by Auwalms — APPROVED** [review]
Nice job

**Pull request merged**